ballista/rust/scheduler/src/state/execution_graph/execution_stage.rs

```diff
@@ -73,6 +73,30 @@ impl Debug for ExecutionStage {
     }
 }
 
+impl ExecutionStage {
+    /// Get the name of the variant
+    pub(crate) fn name(&self) -> &str {
+        match self {
+            ExecutionStage::UnResolved(_) => "Unresolved",
```

Comment:
Should this simply be the `impl Display` of `ExecutionStage`?

Reply:
I renamed this to `variant_name()` to make it clear that this is just returning the variant name, not the full details of the stage